guides/Distribution.md

```diff
@@ -42,6 +42,13 @@ To create a .app on macOS, first get the stock LÖVR.app, either by downloading
 [here](https://lovr.org/download/mac) or by setting the `-DLOVR_BUILD_BUNDLE=ON` flag when building
 with CMake.
 
+:::note
+If you build your own .app bundle, you may need to bundle MoltenVK and Vulkan libraries for
```

Comment:
Maybe instead of "may", it should say "you will need to bundle MoltenVK and Vulkan libraries for distribution.  Otherwise, the .app will only work if the user has MoltenVK installed." or something similar.

Reply:
Actually, I guess the next sentence makes it more clear that it's required, and not really optional.